game-headless/src/main/java/org/triplea/game/server/HeadlessGameRunner.java

```diff
@@ -17,6 +19,13 @@ public static void main(final String[] args) {
   }
 
   private static Injections constructInjections() {
-    return Injections.builder().engineVersion(new ProductVersionReader().getVersion()).build();
+    return Injections.builder()
+        .engineVersion(new ProductVersionReader().getVersion())
+        .playerTypes(gatherPlayerTypes())
+        .build();
+  }
+
+  private static Collection<PlayerTypes.Type> gatherPlayerTypes() {
+    return PlayerTypes.getBuiltInPlayerTypes();
```

Comment:
nit, looks like it would be more concise and consistent (with `  .engineVersion(new ProductVersionReader().getVersion())`) to inline this method

Reply:
@DanVanAtta , I just made that change.